analysis/R/analysis_tool.R

```diff
@@ -16,6 +16,7 @@
 
 library(optparse)
 
+setwd("~/git/google_dev_rappor/analysis/R/")
```

Comment:
Fixed. I thought I took care of all these tweaks, but apparently I missed this one out.